pages/linux/mkfs.md

```diff
@@ -0,0 +1,16 @@
+# mkfs
+
+> Build a Linux filesystem on a hard disk partition.
+> This mkfs frontend is deprecated in favour of filesystem specific mkfs.<type> utils.
+
+- Build a Linux ext2 filesystem on a partition:
+
+`mkfs {{path/to/partition}}`
+
+- Build a filesystem of a specified type:
+
+`mkfs -t {{filesystem_type}} {{path/to/partition}}`
+
+- Build a filesystem of a specified type and check for bad blocks:
+
+`mkfs -c -t {{filesystem_type}} {{path/to/partition}}`
```

Comment:
Some other type here.

Reply:
Do you mean something like this?

- Build a ntfs filesystem and check for bad blocks:

`mkfs -c -t {{ntfs}} {{path/to/partition}}`

Reply:
Yeah, but you don't need to change the description. Just the example.